src/plugins/orangehrmRecruitmentPlugin/entity/Decorator/CandidateDecorator.php

```diff
@@ -65,7 +65,7 @@ public function getStatus(): array
         $status = $this->candidate->getStatus();
         return [
             'id' => $status,
-            'label' => ucwords(strtolower(CandidateAPI::STATUS_MAP[$status]))
+            'label' => ucwords(strtolower(CandidateService::STATUS_MAP[$status]))
         ];
```

Comment:
I missed in a review. usually this is not a job of a decorator. I think this should goes to model or somewhere